src/main/java/ex_1.java

```diff
@@ -0,0 +1,21 @@
+import java.util.Scanner;
+
+public class ex_1 {
```

Comment:
You could rename the class to be more representative of what is done here. For example: SecondsConverter

Reply:
Sure